include/votca/xtp/molden.h

```diff
@@ -0,0 +1,85 @@
+/*
+ *            Copyright 2009-2020 The VOTCA Development Team
+ *                       (http://www.votca.org)
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License")
+ *
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#pragma once
+#ifndef VOTCA_XTP_MOLDEN_H
+#define VOTCA_XTP_MOLDEN_H
+
+// Local VOTCA includes
+#include "votca/xtp/logger.h"
+#include "votca/xtp/orbreorder.h"
+#include "votca/xtp/qmtool.h"
+#include <fstream>
+#include <votca/xtp/orbitals.h>
+
+namespace votca {
+namespace xtp {
+class Molden {
+ public:
+  Molden(Logger& log) : _log(log){};
+
+  ~Molden() = default;
+
+  void WriteFile(const std::string& filename, const Orbitals& orbitals);
```

Comment:
is the function `const`

Reply:
No, it needs to load the basisset from info in the orbital object

Reply:
Orbitals has this nice function, which returns an AOBasis. Then you probably do not have to keep either of them around as member variables.

Reply:
That turns out to be true